web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -611,7 +615,7 @@ const TeamReviews = ({ onBack, periodId }) => {
         },
         data: {
           ...period,
-          reviewStatus: ReviewStatus.AWAITING_APPROVAL
+          reviewStatus: ReviewStatus.OPEN
```

Comment:
When a review period is "launched", its status changes to OPEN, not AWAITING_APPROVAL.